bin/lib/cli-utils.js

```diff
@@ -41,7 +41,22 @@ function loadConfig (program, options) {
     const config = JSON.parse(file)
     argv = { ...config, ...argv }
   } catch (err) {
```

Comment:
You mean don't even bother with whether the user specified a config file or not, just always display the error and exit?  I'd be fine with that as well, I was just allowing for the legacy default behavior to still execute in case someone relied on it.  I certainly think the old behavior is wrong if the user specified a config file, or if the file exists and it couldn't be parsed.  But if nothing is specified and the file doesn't exist - what is the correct behavior?

Reply:
I was more saying that if a file doesn't exist, `err` would be a `file does not exist error` and instead of using `fileExists()`, which takes time to read from the disk, use the error type in the if statement.

But this is such a small optimization it might not matter